ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -473,6 +474,11 @@ message MaximumRecordTimeExceeded {
   string details = 1;
 }
 
+// The ledger time of the transaction submission violates some constraint.
+message InvalidLedgerTime {
+  string details = 1;
```

Comment:
Is a string sufficient here, i.e., wouldn't it be beneficial to store e.g. the exact time that violated a time constraint?

Reply:
All of these messages (e.g., `MaximumRecordTimeExceeded`, `PartyNotKnownOnLedger`, `Inconsistent`) only contain one human-readable description that ultimately ends up in a gRPC error status.

Reply:
I see. In that cases I don't see the point in having separate messages if they contain the same field. I.e., an enum plus one message definition would suffice.